queue-health/graph/graph.py

```diff
@@ -121,13 +121,16 @@ def render(history_lines, out_file):
             did_merge = 0
 
         if online:  # Ignore offline status
-            merge_sum += did_merge
             last_merge = merged
 
         daily_queue.append(happy)
-        daily_merged.append(did_merge)
         if len(daily_queue) > 60*24:
             happy_sum -= daily_queue.popleft()
+
+        if queue or did_merge:  # Only add samples when things are in the queue.
```

Comment:
I really don't like tweaking this number. Let's get the merge queue to emit the stats you want instead of this. There's many cases in which this number isn't going to reflect very well the actual number of daily merges if you make this change. I don't think "daily merges we might have made if we had work to do" is a number that we should even want to try and make up.